ushahidi/platform-client
# 103 cached user values wiped

This pull request makes the following changes:
- Fixes user edit profile not displaying updated values

Test these changes by:
- Select Account Settings from the User menu in the top right hand corner
- Update an aspect of the user profile, user email
- save
- go to home, and return to user profile edit to ensure that the email is still updated
- logout, login and check user profile to make sure email is still updated

Fixes ushahidi/platform# 103

Ping @ushahidi/platform


[<img src="https://reviewable.io/review_button.svg" height="40" alt="Review on Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/116)
